src/SFA.DAS.AssessorService.Application.Api/Services/OuterApiService.cs

```diff
@@ -1,46 +1,48 @@
 ﻿using System;
 using System.Collections.Generic;
-using System.Linq;
 using System.Threading.Tasks;
-using Microsoft.Extensions.Logging;
 using SFA.DAS.AssessorService.Application.Infrastructure.OuterApi;
 using SFA.DAS.AssessorService.Application.Interfaces;
 
 namespace SFA.DAS.AssessorService.Application.Api.Services
 {
     public class OuterApiService : IOuterApiService
     {
-        private readonly IOuterApiClient outerApiClient;
-        private readonly ILogger<OuterApiService> logger;
+        private readonly IOuterApiClient _outerApiClient;
 
-        public OuterApiService(IOuterApiClient outerApiClient, ILogger<OuterApiService> logger)
+        public OuterApiService(IOuterApiClient outerApiClient)
         {
-            this.outerApiClient = outerApiClient;
-            this.logger = logger;
+            this._outerApiClient = outerApiClient;
```

Comment:
unnecessary `this` keyword when using _camelCase

Reply:
Fixed